moveit_setup_assistant/src/widgets/start_screen_widget.cpp

```diff
@@ -437,10 +437,11 @@ bool StartScreenWidget::loadExistingFiles()
   // Load 3d_sensors config file
   load3DSensorsFile();
 
+  // TODO: after removing the robot namespace from ros_controllers.yaml, inputROSControllersYAML needs to be refactored
   // Load ros controllers yaml file if available-----------------------------------------------
-  fs::path ros_controllers_yaml_path = config_data_->config_pkg_path_;
-  ros_controllers_yaml_path /= "config/ros_controllers.yaml";
-  config_data_->inputROSControllersYAML(ros_controllers_yaml_path.make_preferred().native().c_str());
+  // fs::path ros_controllers_yaml_path = config_data_->config_pkg_path_;
```

Comment:
as parsing `ros_controllers.yaml` is not crucial, I think it is better to disable it for now until we get this merged.

Reply:
instead of making a Github comment about this, put this comment in the code so that it isn't lost!